keyboards/easyconvert/info.json

```diff
@@ -0,0 +1,140 @@
+{
+    "keyboard_name": "EASY-CONVERT",
+    "manufacturer": "LiGuo Studio",
+    "url": "",
+    "maintainer": "Cheng Leeren",
+    "usb": {
+        "vid": "0x1433",
+        "pid": "0xC903",
+        "device_version": "0.0.1"
+    },
+    "matrix_pins": {
+        "cols": ["B0", "B1", "B2", "B3", "B4", "B5", "B6", "B7", "B8", "B9", "B10", "B11", "B12", "B13", "B14", "B15"],
+        "rows": ["C0", "C1", "C2", "C3", "C4", "C5"]
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "C7", "pin_b": "C8"}
+        ]
+    },
+    "processor": "STM32L433",
+    "bootloader": "custom",
```

Comment:
"custom" without any reset implementation seems rather incorrect, given the readme suggests use of `QK_BOOT`.

Reply:
This is still outstanding.

Reply:
The bootloader I am using can double click reset to enter the boot, and I have now updated the boot entry method in readme.

Reply:
`QK_BOOT` will not do anything as is. You need to provide an implementation of `bootloader_jump()` if you have set this to `custom`.